### PR TITLE
Makes abductor agent's gun self-recharging.

### PR DESCRIPTION
**Pull request by Arzion5**
:cl:
tweak: Made abductor agent's gun self-recharging
/:cl:

Agents now have less of a hard time managing their side-arm.

### CODE REVIEW

**Comment by KayeArray** [comment]
Isn't the abductor gun already buffed considerably thanks to the new rad system?

**Comment by Arzion5** [comment]
It is only a side-effect of the radiation system rework. Abductor agents required self-rechargeable guns for a long time, especially when they have an infinite-battery omniprod, and there are "abductor batteries" in the code, which are self-rechargeable

**Comment by GuyonBroadway** [comment]
Please limit your PR to one general subject, adding a wizard spell and doing a rebalance means if the headmins likes the spell but disagree with the balance change then the good has to go because its in with the bad.

**Comment by KayeArray** [comment]
ATOMIC

**Comment by Arzion5** [comment]
I still want to leave the voice of god autosurgeon for later though

**Comment by GuyonBroadway** [comment]
you forgot to uninclude the file.

File: hippiestation/code/game/gamemodes/miniantags/abduction/abduction_gear.dm [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/obj/item/gun/energy/alien
+	selfcharge = 1
```
Comment (ghost) [inline hippiestation/code/game/gamemodes/miniantags/abduction/abduction_gear.dm]: TRUE

**Review by ghost — DISMISSED** [review]

**Comment by ghost** [comment]
build this

**Comment by ghost** [comment]
I'm okay with this.

**Review by j-mie — APPROVED** [review]

**Pull request merged**